dashboard/config/routes.rb

```diff
@@ -153,6 +153,7 @@ def redirect_to_teacher_dashboard
   post '/milestone/:user_id/level/:level_id', :to => 'activities#milestone', :as => 'milestone_level'
   post '/milestone/:user_id/:script_level_id', :to => 'activities#milestone', :as => 'milestone'
 
+  get '/admin/monthly', to: 'reports#monthly_metrics', as: 'monthly_metrics'
```

Comment:
we could probably turn this into a wildcard `get /admin/:action` if people didn't like having the urls different from the controller action so much :p